paasta_tools/check_kubernetes_services_replication.py

```diff
@@ -30,27 +30,104 @@
 CRITICAL. If replication_threshold is defined in the yelpsoa config for a service
 instance then it will be used instead.
 """
+import argparse
 import logging
 from typing import Optional
 from typing import Sequence
+from typing import Union
 
+from paasta_tools import eks_tools
 from paasta_tools import kubernetes_tools
 from paasta_tools import monitoring_tools
 from paasta_tools.check_services_replication_tools import main
+from paasta_tools.eks_tools import EksDeploymentConfig
 from paasta_tools.kubernetes_tools import filter_pods_by_service_instance
 from paasta_tools.kubernetes_tools import is_pod_ready
 from paasta_tools.kubernetes_tools import KubernetesDeploymentConfig
 from paasta_tools.kubernetes_tools import V1Pod
 from paasta_tools.long_running_service_tools import get_proxy_port_for_instance
 from paasta_tools.smartstack_tools import KubeSmartstackEnvoyReplicationChecker
+from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import SPACER
 
 
 log = logging.getLogger(__name__)
 DEFAULT_ALERT_AFTER = "10m"
 
 
+def parse_args() -> argparse.Namespace:
```

Comment:
should we import `parse_args` from `paasta_tools/check_services_replication_tools.py` seems like this is a copy of the args otherwise :)